packages/lsp-extension/src/index.ts

```diff
@@ -310,7 +310,7 @@ function addRunningSessionManager(
 
 const adapterTrackerPlugin: JupyterFrontEndPlugin<IWidgetLSPAdapterTracker> = {
   id: '@jupyterlab/lsp-extension:tracker',
-  description: 'Provides the `WidgetLSPAdapter`s tracker.',
+  description: "Provides the WidgetLSPAdapter's tracker.",
```

Comment:
No strong opinion on this one, but if this gets pulled into rst, would a double backtick work?

```suggestion
  description: 'Provides the ``WidgetLSPAdapter``s tracker.',
```

Reply:
Let's try :wink:

Reply:
It seems it does not:

```
/home/runner/work/jupyterlab/jupyterlab/docs/source/extension/plugins_list.rst:118: WARNING: Inline literal start-string without end-string.
```